addon/styles/_frost-button.scss

```diff
@@ -305,6 +297,54 @@ $frost-color-button-in-line-darken-2: darken($frost-color-blue-1, 15);
       box-shadow: none;
     }
   }
+  
+  // design = 'app-bar'
+  &.app-bar {
```

Comment:
Just as a side note, the app bar is nice and clean now, each item is defined as follows:

```{{frost-button
        class='navigation'
        design='app-bar'
        icon='icons/app/nav-network'
        text=(t 'application-bar.network')
        onClick=(action 'openNetworkNavigation')
      }}```

And it used to be like this:

       ```{{#frost-button class='navigation' on-click=(action 'openNetworkNavigation')}}
            <div class='icon-text'>
              {{frost-svg path='app/nav-network'}}
                <div class='text'>{{t 'application-bar.network'}}</div>
            </div>
        {{/frost-button}}```